Code/GraphMol/MolDraw2D/MolDraw2DSVG.cpp

```diff
@@ -355,11 +362,11 @@ void MolDraw2DSVG::addMoleculeMetadata(const ROMol &mol, int confId) const {
     } else {
       dpos = getDrawCoords(dpos);
     }
-    d_os << " drawing-x=\"" << dpos.x << "\""
-         << " drawing-y=\"" << dpos.y << "\"";
-    d_os << " x=\"" << pos.x << "\""
-         << " y=\"" << pos.y << "\""
-         << " z=\"" << pos.z << "\"";
+    d_os << " drawing-x=\"" << formatDouble(dpos.x) << "\""
+         << " drawing-y=\"" << formatDouble(dpos.y) << "\"";
+    d_os << " x=\"" << formatDouble(pos.x) << "\""
+         << " y=\"" << formatDouble(pos.y) << "\""
+         << " z=\"" << formatDouble(pos.z) << "\"";
```

Comment:
The coordinates of the atom itself should not be rounded (or at least not to 0.1):
```suggestion
    d_os << " x=\"" << pos.x << "\""
         << " y=\"" << pos.y << "\""
         << " z=\"" << pos.z << "\"";
```